sonarts-sq-plugin/its/plugin/src/test/java/org/sonar/typescript/its/ConfigPathTest.java

```diff
@@ -82,9 +82,9 @@ public void withWrongTSconfigPathTest() {
     request.setComponentKeys(Collections.singletonList(PROJECT_KEY)).setRules(ImmutableList.of("typescript:S1764"));
     List<Issues.Issue> issuesList = newWsClient().issues().search(request).getIssuesList();
 
-    assertThat(issuesList).hasSize(1);
-    assertThat(buildResult.getLogs()).contains("Check property specified in sonar.typescript.tsconfigPath");
-    assertThat(buildResult.getLogs()).contains("No tsconfig.json file found");
+    // TODO should we analyze if configured tsconfig doesn't exists
```

Comment:
I don't get the point of this todo

Reply:
Before analysis was analyzing even if `tsconfig` provided didn't exist. It would use default tsconfig in such case. Now we don't have this mechanism anymore

Reply:
TBH I'm not sure about the best strategy here. but we should think about it when we will add support for analysis without tsconfig.